packages/dapi/lib/grpcServer/handlers/blockheaders-stream/subscribeToBlockHeadersWithChainLocksHandlerFactory.js

```diff
@@ -75,10 +75,15 @@ function subscribeToBlockHeadersWithChainLocksHandlerFactory(
   async function subscribeToBlockHeadersWithChainLocksHandler(call) {
     const { request } = call;
 
-    let fromBlockHash = Buffer.from(request.getFromBlockHash()).toString('hex');
+    const fromBlockHash = Buffer.from(request.getFromBlockHash()).toString('hex');
     const fromBlockHeight = request.getFromBlockHeight();
+    const from = fromBlockHash || fromBlockHeight;
     const count = request.getCount();
 
+    if (from === 0) {
```

Comment:
This chek little bit confusing from readability and understanding point of view.
Could we make this before we create `from` variable? Something like:

```js
     const fromBlockHashHex = Buffer.from(request.getFromBlockHash()).toString('hex');
     const fromBlockHeight = request.getFromBlockHeight();
     
     if (fromBlockHashHex === '' && fromBlockHeight === 0) {
        throw new InvalidArgumentGrpcError('Minimum value for `fromBlockHeight` is 1');
     }
     
     const from = fromBlockHash || fromBlockHeight;
```